<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑防护措施</string>
  </property>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>410</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;.SF NS Text&quot;;
background-color: rgb(98, 160, 226);</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="edit_input5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>601</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Adobe Arabic&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;.SF NS Text&quot;;</string>
   </property>
   <property name="text">
    <string>内容详情：</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>601</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
    <item row="0" column="1">
     <widget class="QComboBox" name="edit_input1">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;.SF NS Text&quot;;</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QComboBox" name="edit_input2">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;.SF NS Text&quot;;</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_type_second">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;.SF NS Text&quot;;
color: ;</string>
      </property>
      <property name="text">
       <string>防护措施类别：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_type_first">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;.SF NS Text&quot;;
color: ;</string>
      </property>
      <property name="text">
       <string>防护对象类别：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_type_third">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;.SF NS Text&quot;;</string>
      </property>
      <property name="text">
       <string>防护手段：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edit_input3">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;.SF NS Text&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="edit_input4">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;.SF NS Text&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_type_fourth">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;.SF NS Text&quot;;</string>
      </property>
      <property name="text">
       <string>行业类型：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>410</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;.SF NS Text&quot;;
background-color: rgb(240, 252, 252);</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
